src/components/widgets/WebWidget.tsx

```diff
@@ -0,0 +1,39 @@
+import { useNavigation } from '@react-navigation/core';
+import React, { useCallback } from 'react';
+
+import { Icon, normalize, texts } from '../../config';
+import { ScreenName, WidgetProps } from '../../types';
+import { Image } from '../Image';
+
+import { DefaultWidget } from './DefaultWidget';
+
+export const WebWidget = ({ text, additionalProps }: WidgetProps) => {
+  const navigation = useNavigation();
+
+  const onPress = useCallback(() => {
+    navigation.navigate(ScreenName.Web, {
+      webUrl: additionalProps?.webUrl,
+      title: additionalProps?.staticContentTitle ?? texts.widgets.web
+    });
+  }, [navigation, text]);
+
+  return (
+    <DefaultWidget
+      Icon={() =>
+        additionalProps?.icon ? (
+          <Image
```

Comment:
wouldn't this be a nice feature for every widget? having an image instead of icons can be part of the default widget component, or am i wrong?

Reply:
Yes that would be a nice feature for all widgets. But I prefer to create a new ticket for this feature, adding this feature is not included in this ticket.

Reply:
no need for a new ticket as discussed with @juliahebestreit

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/commit/f34f4e3e5b22bdb3f7fa60daa5af81bfd3d70eb3